<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>parseWizard</class>
 <widget class="QWizard" name="parseWizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>442</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wizard</string>
  </property>
  <widget class="QWizardPage" name="wizardPage1">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>250</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="MyResources.qrc">:/images/assist.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="1" alignment="Qt::AlignTop">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>This wizard will help you through the parsing process .You can input the text needed for the parser, by manually typing copy and pasting, or uploading an input file.A default label will be assigned to the output, which is editable upon preference.</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>INTRODUCTION</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="wizardPage2">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Article Title:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="1" column="1">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0" alignment="Qt::AlignLeft|Qt::AlignTop">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Author:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QTableView" name="tableView">
        <property name="maximumSize">
         <size>
          <width>12000</width>
          <height>300</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="3" column="1" alignment="Qt::AlignHCenter">
       <widget class="QPushButton" name="pushButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>600</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>600</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Add Author</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="wizardPage3">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="5" column="0">
     <widget class="QLineEdit" name="lineEdit_2"/>
    </item>
    <item row="8" column="0" rowspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Volume: </string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignLeft">
       <widget class="QComboBox" name="comboBox"/>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Issue:</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignLeft">
       <widget class="QComboBox" name="comboBox_2"/>
      </item>
     </layout>
    </item>
    <item row="6" column="0">
     <widget class="QTableView" name="tableView_2"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Journal:</string>
      </property>
     </widget>
    </item>
   </layout>
   <zorder>lineEdit_2</zorder>
   <zorder>tableView_2</zorder>
   <zorder>horizontalLayoutWidget_2</zorder>
   <zorder>label_5</zorder>
  </widget>
 </widget>
 <resources>
  <include location="MyResources.qrc"/>
 </resources>
 <connections/>
</ui>
